doc/classes/Projection.xml

```diff
@@ -40,12 +47,14 @@
 			<return type="Projection" />
 			<param index="0" name="flip_y" type="bool" />
 			<description>
+				Creates a new [Projection] that projects positions with an adjusted depth of field, and flips the projected positions vertically, according to [param flip_y].
```

Comment:
```suggestion
				Creates a new [Projection] that projects positions from a depth range of [code]-1[/code] to [code]1[/code] to one that ranges from [code]0[/code] to [code]1[/code], and flips the projected positions vertically, according to [param flip_y].
```

Reply:
Funny enough, double checking this one caused me to spot a bug in the OpenGL renderer. I had used this completely wrong :stuck_out_tongue:

Reply:
Good docs ~~save lifes~~ prevent bugs! :D